api/clients/v2/mock/retrieval_client.go

```diff
@@ -16,8 +17,28 @@ type MockRetrievalClient struct {
 	mock.Mock
 }
 
+func (_m *MockRetrievalClient) GetBlob(
+	ctx context.Context,
+	blobKey v2.BlobKey,
+	blobVersion uint16,
+	blobCommitments encoding.BlobCommitments,
+	referenceBlockNumber uint64,
+	quorumID uint8,
+) ([]byte, error) {
+	return _m.GetBlobWithProbe(ctx, blobKey, blobVersion, blobCommitments, referenceBlockNumber, quorumID, nil)
+}
+
 // GetBlob provides a mock function with given fields: ctx, blobKey, blobVersion, blobCommitments, referenceBlockNumber, quorumID
-func (_m *MockRetrievalClient) GetBlob(ctx context.Context, blobKey v2.BlobKey, blobVersion uint16, blobCommitments encoding.BlobCommitments, referenceBlockNumber uint64, quorumID uint8) ([]byte, error) {
+func (_m *MockRetrievalClient) GetBlobWithProbe(
```

Comment:
mock needs to be regenerated with most recent changes

Reply:
regenerated